test/index.spec.js

```diff
@@ -59,6 +59,22 @@ it("should hash palette based pngs correctly", async () => {
   expect(h1).toBe(h2);
 });
 
+/**
+ * For some jpeg images, cwasm-jpeg-turbo throws longjmp error due to a bug in WebAssembly.
+ * Added a workaround to use alternate libraries to read such images.
+ *
+ * see: https://github.com/LinusU/cwasm-jpeg-turbo/issues/2
+ * see: https://github.com/pwlmaciejewski/imghash/issues/21
+ */
+it("should not throw longjmp error", () => {
```

Comment:
Ok @bbhopesh, I see why it reports some other test failing.
Async function will be executed after the test execution. Or in other words, async/await doesn't go well with `.toThrow()`. Here's a general solution: https://github.com/facebook/jest/issues/1700#issuecomment-377890222

I would rewrite this test to something like this:

```js
it("should not throw longjmp error",  () => {
  expect(imghash.hash(__dirname + "/files/longjmperror.jpg", 8)).resolves.toHaveLength(16)
});
```